base/src/test/java/io/spine/type/OneofDeclarationTest.java

```diff
@@ -18,14 +18,13 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.code.gen.java;
+package io.spine.type;
```

Comment:
The same question regarding the package as for the class under the test. Why do change the package?

Reply:
We move `OneofDeclaration` from another module. I've changed the package to a more appropriate `io.spine.code.proto`, which is also the package where the `FieldDeclaration` resides.